test/jdk/java/io/FileOutputStream/ManyFiles.java

```diff
@@ -37,14 +38,12 @@ public class ManyFiles {
     static int NUM_FILES = 2050;
 
     public static void main(String args[]) throws Exception {
+        // Vestigial comment from before @requires tag was added:
         // Linux does not yet allow opening this many files; Solaris
```

Comment:
Nit - If you do happen to update this PR for any other reason, then maybe consider adding jtreg's `@comment` in the test definition (https://openjdk.org/jtreg/tag-spec.html#INFORMATIONAL_TAGS) just before the `@requires` line and move this existing comment there.

Reply:
> then maybe consider adding jtreg's `@comment` in the test

Added in fb9f0c5.